src/common/error/internalErrorCode.ts

```diff
@@ -44,6 +44,7 @@ export enum InternalErrorCode {
         RNTempFolderDeletionFailed = 707,
         DebuggingFailedInNodeWrapper = 708,
         PlatformNotSupported = 708,
```

Comment:
I don't know if this was intended, but `DebuggingFailedInNodeWrapper` and `PlatformNotSupported` have the same error code